content/applications/services/helpdesk/overview/receiving_tickets.rst

```diff
@@ -4,274 +4,291 @@ Start receiving tickets
+--> Helpdesk Teams`. Find the :guilabel:`Website Form` feature under the :guilabel:`Help Center`
+section, and check the box. If more than one website is active on the database, confirm the correct
+website is listed in the :guilabel:`Website` field. If not, select the correct one from the
+drop-down list.
+
+After the feature is activated, click the :guilabel:`Go to Website` smart button at the top of the
+:guilabel:`Teams` settings page to view and edit the new website form, which is created
+automatically by Odoo.
+
+.. note::
+   After enabling the website form, the *Teams* settings page may need to be refreshed before the
+   *Go to Website* smart button appears.
+
+   As well, if a *Help Center* is published, the smart button navigates there first. Simply click
+   the :guilabel:`Contact Us` button at the bottom of the forum to navigate to the ticket
+   submission form.
+
+.. image:: receiving_tickets/receiving-tickets-go-to-website.png
+   :align: center
+   :alt: View of the settings page of a helpdesk team emphasizing the Go to Website button in
+         Odoo Helpdesk.
+
+Customize the website ticket form
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To customize the default ticket submission form, click the :guilabel:`Edit` button in the upper
+right corner of the page. This opens the editing pane on the right side of the window. Click on one
+of the fields in the form to edit.
+
+To add a new field, go to the :guilabel:`Field` section and click :guilabel:`+Field`. Click the
+:guilabel:`🗑️ (trash can)` icon to delete the field. Edit the other options for the new field as
+necessary:
+
+- :guilabel:`Type`: which matches an Odoo model value to the field (e.g. `Customer Name`).
+- :guilabel:`Input Type`: to determine what type of input the field should be, like `Text`, `Email`,
+  `Telephone` or `URL`.
+- :guilabel:`Label`: to give the form field a label (e.g. `Full Name`, `Email Address`, etc.). Also
+  control the label position on the form by using the nested :guilabel:`Position` options.
+- :guilabel:`Description`: which, optionally, adds an editable line under the input box to provide
+  additional contextual information related to the field.
+- :guilabel:`Placeholder`: to add a sample input value.
+- :guilabel:`Default value`: to add common use case values that most customers would find valuable.
+  For example, this can include prompts of information customers should include to make it easier to
+  solve their issue, such as an account number, or product number.
+- :guilabel:`Required`: to mark a field as required in order for the form to be submitted, toggle
+  the switch from gray to blue.
+- :guilabel:`Visibility`: to allow for absolute or conditional visibility of the field. Nested
+  options, such as device visibility, appear when certain options are selected.
+- :guilabel:`Animation`: choose whether or not the field should include animation.
```

Comment:
```suggestion
- :guilabel:`Type`: matches an Odoo model value to the field (e.g. `Customer Name`).
- :guilabel:`Input Type`: determine what type of input the field should be, like `Text`, `Email`,
  `Telephone` or `URL`.
- :guilabel:`Label`: give the form field a label (e.g. `Full Name`, `Email Address`, etc.). Also
  control the label position on the form by using the nested :guilabel:`Position` options.
- :guilabel:`Description`: determine whether or not to add an editable line under the input box to
  provide additional contextual information related to the field.
- :guilabel:`Placeholder`: add a sample input value.
- :guilabel:`Default value`: add common use case values that most customers would find valuable.
  For example, this can include prompts of information customers should include to make it easier to
  solve their issue, such as an account number, or product number.
- :guilabel:`Required`: determine whether or not to mark a field as required, in order for the form
  to be submitted. Toggle the switch from gray to blue.
- :guilabel:`Visibility`: allow for absolute or conditional visibility of the field. Nested options,
  such as, device visibility, appear when certain options are selected.
- :guilabel:`Animation`: choose whether or not the field should include animation.
```